.github/workflows/sbom.yaml

```diff
@@ -19,9 +19,9 @@ jobs:
         run: |
           TAG=${{ github.event.inputs.tag }}
           echo "TAG=$TAG" >> "$GITHUB_ENV"
-      - uses: actions/setup-go@cdcb36043654635271a94b9a6d1392de5bb323a7 # v3
+      - uses: actions/setup-go@cdcb36043654635271a94b9a6d1392de5bb323a7
```

Comment:
As mentioned here, the dependabot can not modify comments. 
So, I removed it.

https://github.com/kubernetes-sigs/kueue/pull/2298#discussion_r1617717296